### ISSUE TITLE
Validate v1 and v2 charts for  RHEL 7.9

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#30887]
Validated V2 charts for RHEL 7.9 Selinux enabled clusters.

**Comment by aaronRancher** [comment]
The following v2 charts have been validated on Rancher `v2.5.5`. Installs and P0 use cases pass on an RHEL 7.9 install with Selinux enabled.

OPA Gatekeeper: v3.2.101
Monitoring: v9.4.202
Istio: v1.7.301
CIS: v1.0.300
Logging: v3.8.201
Longhorn: v1.1.000

**Comment by aaronRancher** [comment]
The following v1 charts are validated with `v2.5.5`. Installs and P0 use cases pass on an RHEL 7.9 install with Selinux enabled.
Monitoring: v0.2.0
CIS: RKE-CIS-1.5
Pipelines: pipeline-tools:v0.1.15
Alerts: cluster-alerting:v0.2.0
Logging: fluentd:v0.1.22
Istio 1.5.9

The UI has a warning that the following charts have been depreciated as of Rancher `v2.5.0`
Istio
Alerts
Logging
Monitoring
Pipelines

Ex:
![Screen Shot 2021-01-20 at 2 29 06 PM](https://user-images.githubusercontent.com/57409052/105247149-8bafa000-5b31-11eb-91f2-a9394efda0a0.png)

**Comment by aaronRancher** [comment]
Currently blocked on validating Logging: `fluentd:v0.1.22`
https://github.com/rancher/rancher/issues/30949

**Comment by aaronRancher** [comment]
Will continue to track Logging v2 for RHEL 7.9 in a separate issue:
https://github.com/rancher/rancher/issues/30949

Closing this issue.
